bundle/bin/rke2-uninstall.sh

```diff
@@ -35,10 +35,9 @@ uninstall_disable_services()
 uninstall_remove_files()
 {
     if [ -r /etc/redhat-release ] || [ -r /etc/centos-release ] || [ -r /etc/oracle-release ]; then
-        rm -f /etc/yum.repos.d/rancher-rke2*.repo
-        rm -f /etc/sysconfig/rke2-server
-
         yum remove -y rke2-*
```

Comment:
While we're in here, could we pull https://github.com/rancher/rke2/pull/512 (quoting `rke2-*` to prevent globbing) into this script?